widgets/blankcanvas/config.ts

```diff
@@ -0,0 +1,98 @@
+import { ExpandedTileOptions, InlineTileOptions, Style } from "packages/widget-utils/dist/esm"
+
+export const config: {
+  style: Style
```

Comment:
Blank canvas does not have these settings in the backend, so the user defines them from the frontend